### PR TITLE
O365 - Security And Compliance - Validate sc-auth-start run before sc-auth-complete

### PR DESCRIPTION
**Pull request by GalRabin**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/32274

##### O365 - Security And Compliance - Content Search (beta)
- Integration setup error handle - Validate !o365-sc-auth-start run before !o365-sc-auth-complete

## Screenshots
image

## Minimum version of Demisto
- [ ] 5.0.0
- [x] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/EWS/Integrations/SecurityAndCompliance/SecurityAndCompliance.ps1 [hunk 1]
```diff
@@ -495,6 +495,10 @@ class OAuth2DeviceCodeClient {
     }
 
     [psobject]AccessTokenRequest() {
+        # Verify that user run start before complete
+        if (!$this.device_code) {
+            throw "Please run !o365-sc-auth-start and follow the command instructions"
```
Comment (yaakovi) [inline Packs/EWS/Integrations/SecurityAndCompliance/SecurityAndCompliance.ps1:500]:
```suggestion
            throw "Please run !o365-sc-auth-start and follow the command instructions."
```

File: Packs/EWS/pack_metadata.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
     "name": "EWS",
     "description": "Exchange Web Services and Office 365 (mail)",
     "support": "xsoar",
-    "currentVersion": "1.6.1",
+    "currentVersion": "1.7.0",
```
Comment (yaakovi) [inline Packs/EWS/pack_metadata.json:5]: a bug fix should be a revision, not minor
```suggestion
    "currentVersion": "1.6.2",
```

File: Packs/EWS/ReleaseNotes/1_6_2.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### O365 - Security And Compliance - Content Search (beta)
+- Integration setup error handle - Validate !o365-sc-auth-start run before !o365-sc-auth-complete
```
Comment (yaakovi) [inline Packs/EWS/ReleaseNotes/1_6_2.md:4]: @amshamah419 what do you think?
```suggestion
Improved the integration setup error handling - validate the user run the ***o365-sc-auth-start*** command before running the ***o365-sc-auth-complete*** command.
```

**Review by yaakovi — APPROVED** [review]

File: Packs/EWS/ReleaseNotes/1_6_2.md (hunk 3 above)
Comment (amshamah419) [reply Packs/EWS/ReleaseNotes/1_6_2.md]: run -> ran and it should be good to go

File: Packs/EWS/ReleaseNotes/1_6_2.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### O365 - Security And Compliance - Content Search (beta)
+Improved the integration setup error handling - validate the user run the ***o365-sc-auth-start*** command before running the ***o365-sc-auth-complete*** command.
```
Comment (amshamah419) [inline Packs/EWS/ReleaseNotes/1_6_2.md:4]:
```suggestion
Improved the integration setup error handling. Now validates that the user ran the ***o365-sc-auth-start*** command before running the ***o365-sc-auth-complete*** command.
```

**Pull request merged**